packages/terra-form-select/src/SearchSelect.jsx

```diff
@@ -230,6 +235,7 @@ class SearchSelect extends React.Component {
         totalOptions={SelectUtil.getTotalNumberOfOptions(children)}
         clearOptionDisplay={clearOptionDisplay}
         inputId={inputId}
+        ariaLabel={ariaLabel}
```

Comment:
I think you should use ariaLabel() function here instead of assigning with direct prop to prevent unexpected SR responses.